tests/cypress/integration/reportRepair/confirmation.spec.js

```diff
@@ -1,97 +1,22 @@
 import {
+  completeJourney,
   intercept_address_search,
   intercept_availability_search,
   intercept_repair_triage,
   intercept_save_repair,
```

Comment:
We've got inconsistent name casing. Are these all helper functions for the cypress tests alone? Perhaps we can update their name casing?

Reply:
Yes, these are all Cypress test helpers.

To keep the changes in this PR focused, making names consistent will be part of another PR.